com.gitlab.newsflash.json

```diff
@@ -47,8 +47,8 @@
             "sources" : [
                 {
                     "type" : "git",
-                    "url" : "https://source.puri.sm/Librem5/libhandy",
-                    "tag": "v0.0.13"
+                    "url" : "https://gitlab.gnome.org/GNOME/libhandy",
+                    "tag": "1.0.1"
```

Comment:
No, such bump isn't possible. As the API changed between 0.0.13 & 1.0.0.

Reply:
Yup, master has been ported to libhandy1 already. But 1.0.5 doesn't support it.

Reply:
I see, I'll revert it